client/layout/activity-panel/activity-card/test/index.js

```diff
@@ -0,0 +1,100 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import Gridicon from 'gridicons';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import ActivityCard from '../';
+import Gravatar from 'components/gravatar';
+
+describe( 'ActivityCard', () => {
+	test( 'should have correct title', () => {
+		const card = <ActivityCard title="Inbox message">This card has some content</ActivityCard>;
+		expect( card.props.title ).toBe( 'Inbox message' );
+	} );
+
+	test( 'should render a basic card', () => {
+		const card = shallow(
+			<ActivityCard title="Inbox message">This card has some content</ActivityCard>
+		);
+		expect( card ).toMatchSnapshot();
+	} );
+
+	test( 'should render an unread bubble on a card', () => {
+		const card = shallow(
+			<ActivityCard title="Inbox message" unread>
+				This card has some content
+			</ActivityCard>
+		);
+		expect( card ).toMatchSnapshot();
+	} );
+
+	test( 'should render a custom icon on a card', () => {
+		const card = shallow(
+			<ActivityCard title="Inbox message" icon={ <Gridicon icon="customize" /> }>
+				This card has some content
+			</ActivityCard>
+		);
+		expect( card ).toMatchSnapshot();
+	} );
+
+	test( 'should render a gravatar on a card', () => {
+		const card = shallow(
+			<ActivityCard title="Inbox message" icon={ <Gravatar user="admin@local.test" /> }>
+				This card has some content
+			</ActivityCard>
+		);
+		expect( card ).toMatchSnapshot();
+	} );
+
+	xtest( 'should render a timestamp on a card', () => {
```

Comment:
I'd like to see if we could leave this test in, but the snapshot stays fixed ("3 days ago") while the actual output of the component changes based on date. Could we mock the date formatter? Would probably want to address in a follow-up PR

Reply:
Could we create a timestamp on the fly in the test here using moment so the date is 3 days ago from today?